src/vs/editor/contrib/codeAction/browser/codeActionCommands.ts

```diff
@@ -231,6 +231,14 @@ export class OrganizeImportsAction extends EditorAction {
 				primary: KeyMod.Shift | KeyMod.Alt | KeyCode.KeyO,
 				weight: KeybindingWeight.EditorContrib
 			},
+			metadata: {
+				description: "Organize imports by removing unused imports, sorting imports and combining imports from the same module",
+				args: [{
```

Comment:
This command doesn't take any arguments. There are actually separate commands for `sort` imports